lib/dbservice/profile/teacher_profile.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Dbservice.Profiles.TeacherProfile do
+  @moduledoc false
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Dbservice.Profiles.UserProfile
+  alias Dbservice.Users.Teacher
+
+  schema "teacher_profile" do
+    field(:teacher_id, :string)
+    field(:school, :string)
+    # should i go with program_manager_id instead
+    field(:program_manager, :string)
```

Comment:
Yes. Let's go with the `program_manager_id` instead and remove the comment

Reply:
removing comment now.. i'll go for `program_manager_id` and `school_id` etc in next PR once i complete code for this. my idea is to have fewer references to other tables in profile. so that no additional request is to be made on frontend like gurukul.